src/models/FieldLayoutTab.php

```diff
@@ -54,7 +54,10 @@ public static function createFromConfig(array $config): self
     public function labelHtml(): string
     {
         return
-            Html::tag('span', Html::encode($this->name)) .
+            Html::tag('span', Html::encode($this->name), [
+                'role' => 'heading',
+                'aria-level' => '3',
+            ]) .
```

Comment:
@gcamacho079 Any reason not to just make this an `<h3>` tag?

Reply:
@brandonkelly I can definitely make it an `<h3>` tag. This method required no CSS changes so I thought I’d give it a try that way. 👍🏼